taxonomies/food/ingredients.txt

```diff
@@ -57744,6 +57764,11 @@ carbon_footprint_fr_foodges_value:fr: 0.5
 ciqual_food_code:en: 20047
 ciqual_food_name:en: Tomato, raw
 ciqual_food_name:fr: Tomate, crue
+ecobalyse:en: tomato-organic-eu
```

Comment:
The default should be the "Tomate UE Conv" I think.

Reply:
I do agree with you ! However 'tomato-eu' is not yet available :/